input/templates/partials/catalog/search-result.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="search-result">
+  <div class="search-results-title">{{i18n "catalog:searchResult.title"}}</div>
+  <div class="search-result">{{search-result.keyword}}</div>
```

Comment:
Don't use non-alphanumerical characters in the path for Hbs, otherwise we won't be able to implement it in Java. Use `searchResult` instead.